modules/apps/product-navigation/product-navigation-simulation-web/src/main/java/com/liferay/product/navigation/simulation/web/internal/product/navigation/control/menu/SimulationProductNavigationControlMenuEntry.java

```diff
@@ -233,6 +233,7 @@ private void _processBodyBottomTagBody(PageContext pageContext) {
 
 			IconTag iconTag = new IconTag();
 
+			iconTag.setAriaLabel("Close");
```

Comment:
Hey @orsolyaDekany, I think it should be Intl as [`_language.get(httpServletRequest, "simulation")`](https://github.com/liferay-frontend/liferay-portal/blob/d996827a5f3cd079e3f994e62a4f83441c8ca0da/modules/apps/product-navigation/product-navigation-simulation-web/src/main/java/com/liferay/product/navigation/simulation/web/internal/product/navigation/control/menu/SimulationProductNavigationControlMenuEntry.java#L153) in this LOC